airflow/dags/pre_sync_documents_to_kernel.py

```diff
@@ -31,7 +31,8 @@
 default_args = {
     "owner": "airflow",
     "depends_on_past": False,
-    "start_date": datetime.utcnow(),
+    "start_date": datetime(2019, 8, 21),
```

Comment:
isso é fixo mesmo?

Reply:
@robertatakenaka ate onde sabemos, o melhor eh deixar alguyma data fixa mesmo, 
https://airflow.apache.org/faq.html#what-s-the-deal-with-start-date